src/arm64/simd.h

```diff
@@ -61,8 +144,13 @@ namespace simd {
     // We return uint32_t instead of uint16_t because that seems to be more efficient for most
     // purposes (cutting it down to uint16_t costs performance in some compilers).
     really_inline uint32_t to_bitmask() const {
-      const uint8x16_t bit_mask = {0x01, 0x02, 0x4, 0x8, 0x10, 0x20, 0x40, 0x80,
-                                   0x01, 0x02, 0x4, 0x8, 0x10, 0x20, 0x40, 0x80};
+#ifdef SIMDJSON_REGULAR_VISUAL_STUDIO
+      const uint8x16_t bit_mask =  make_uint8x16_t(0x01, 0x02, 0x4, 0x8, 0x10, 0x20, 0x40, 0x80,
+                                                   0x01, 0x02, 0x4, 0x8, 0x10, 0x20, 0x40, 0x80);
+#else
+      const uint8x16_t bit_mask =  {0x01, 0x02, 0x4, 0x8, 0x10, 0x20, 0x40, 0x80,
```

Comment:
Not a change request at all, but does make_uint8x16_t() generate optimal code on other platforms too, so we wouldn't have to ifdef?

Reply:
> Not a change request at all, but does make_uint8x16_t() generate optimal code on other platforms too, so we wouldn't have to ifdef?

I think it does but I am not sure. I was getting performance warnings so I got nervous and decided to go full on paranoid.